name.md

```diff
@@ -29,3 +29,4 @@ Add Your GitHub Username and GitHub Profile Link
 - [Isembart](https://github.com/Isembart)
 - [Parakrant](https://github.com/Parakrant)
 - [mele000](https://github.com/mele000)
+- [nehalahmedqureshi][https://github.com/NehalAhmedQureshi/onmouse-assignment-]
```

Comment:
```suggestion
- [nehalahmedqureshi][https://github.com/NehalAhmedQureshi]
```
Thank you for the pull request. Just a small change, you need to mention your GitHub profile, not the repository.